recipes/README.md

```diff
@@ -0,0 +1,54 @@
+# 🍝 Recipes
+
+This directory contains recipes that can be used to create templates and starters.
+
+## Templates vs. Starters
+
+- **Templates** are used to create **new exercises, challenges, etc. directly in this repository** using the `npm run create` command.
+- **Starters** are used to create **new projects outside of this repository** using the `npx ghcd neuefische/web-exercises/starters/<starter> my-app -i` command.
+
+The main difference content-wise is that templates `README.md` file contains an instructions template for the exercise, while starters `README.md` file is very bare-bones. See [How?](#how) for more details.
+
+> **Warning**  
+> Never edit the files in the `starters` and `templates` directories directly. Instead, edit the recipe in this folder and then [cook it using the `npm run cook <recipe>` command](#cooking-a-recipe).
+
+## Why?
+
+Basic templates and starters only contain files. But we want some more advanced variants for Create React App and Next.js that are created via automation. The `cook` script automates the creation of all templates and starters from recipes.
+
+## How?
```

Comment:
This part is quite clear, I think.

Regarding the "why" again: sounds like the only reason is that the `readme.md` is different in templates and starters (and other files). We should not repeat other reasons (if there are some), but I think we should be a little bit more clear about our "why" in the respective section above.